keylime/tpm/tpm1.py

```diff
@@ -739,8 +739,8 @@ def check_deep_quote(self, agent_id, nonce, data, quote, vAIK, hAIK, vtpm_policy
                 pcrs.append(line)
 
         # don't pass in data to check pcrs for physical quote
-        return self.check_pcrs(agent_id, tpm_policy, pcrs, None, False, None, None, None) and \
-            self.check_pcrs(agent_id, vtpm_policy, vpcrs, data, True, ima_measurement_list, allowlist, None)
+        return self.check_pcrs(agent_id, tpm_policy, pcrs, None, False, None, None, None, None, {}) and \
```

Comment:
be good to understand the empty dicts again

Reply:
in the subsequent PRs, it will be changed to the "measured boot reference state" supplied by the tenant CLI